### ISSUE TITLE
Mixed Layer Depth

### ISSUE THREAD

**Issue by tylerrohr22** [hrsdawson/ACCESS-WOMBAT_01deg_BGC_validation#4]
1. Validating MLD will be key to looking at any BGC cycling. Some options for reanalysis products to compare to:
-        HYCOM.  Available on the OSU site (http://orca.science.oregonstate.edu/1080.by.2160.monthly.hdf.mld125.hycom.php)
-        bSOSE/SOSE (http://www.ecco.ucsd.edu/sose.html) 

2. Some metrics to look at
-        Winter mean/maximum; summer mean/maxima; timing for onset of shoal

**Comment by hrsdawson** [comment]
@lidefi87 is this something you've already explored?

**Comment by lidefi87** [comment]
Hi Hannah! Yes, I have done this [here](https://github.com/lidefi87/ACCESS_scripts_DFA/blob/main/Scripts/MixedLayerDepth.ipynb). For observations in the Southern Ocean, I used [Sallee 2021](https://www.nature.com/articles/s41586-021-03303-x).

**Comment by hrsdawson** [comment]
Cool, that looks great. Was this for cycle 4 with the BGC?

**Comment by lidefi87** [comment]
No, this was cycle 2, but @TamaraLSchlosser and I will be looking into updating this in the next week or so

**Comment by hrsdawson** [comment]
From meeting today - would be good to look at summertime MLDs. Are they shallower than expected and could this explain the lower productivity and lower iron uptake that we see?

**Comment by lidefi87** [comment]
Sorry I could not make it yesterday. I will likely need to do this for my second chapter anyway, but it will likely focus on the SO. @TamaraLSchlosser, could you confirm what was your source for observations?

**Comment by TamaraLSchlosser** [comment]
I have been looking at a gridded argo product (httpmixedlayer.ucsd.edu) and WOA18. Note these have different definitions of 'mld'. I've uploaded my initial monthly figures to the 'figures' folder but here is a Jan and July below.
**Jan**
![access_argo_woa_mld_7](https://user-images.githubusercontent.com/83566365/207744841-7339e829-8717-4b7c-8bea-dc3cff2a7d73.jpg)
 **July**
![access_argo_woa_mld_1](https://user-images.githubusercontent.com/83566365/207744829-8648e1a5-7bcb-44f7-a849-3c3510067a81.jpg)

**Comment by aekiss** [comment]
MLD comparisons are tricky due to differing definitions and averaging methods - see [Treguier et al 2023](https://doi.org/10.5194/egusphere-2023-310). These issues were dealt with in this preprint, and ACCESS-OM2-01 shows a deep bias in MLD in both summer (top fig below) and winter (bottom fig; different scale) in most of the Southern Ocean and Antarctic shelf which looks consistent with what @TamaraLSchlosser has plotted. In the figs from the preprint below, "ACCESS-MOM" means ACCESS-OM2 at low resolution and ACCESS-OM2-01 at high resolution (IAF cycle 1).

![Screen Shot 2023-03-02 at Thu 2-3 5 05pm 1](https://user-images.githubusercontent.com/31054815/222344985-6117ef3b-ba2a-4c00-a584-4ae1c5c38c8f.png)
![Screen Shot 2023-03-02 at Thu 2-3 5 05pm](https://user-images.githubusercontent.com/31054815/222345000-5bc10540-0e55-4ff4-b050-9e2c4f59456b.png)

**Comment by aekiss** [comment]
[Treguier et al 2023](https://doi.org/10.5194/egusphere-2023-310) mostly use [de Boyer Montégut (2022)](https://doi.org/10.17882/91774) as an observational reference

**Comment by aekiss** [comment]
Also see this forum post: https://forum.access-hive.org.au/t/bgc-validation-access-om2-1-10-degree-iaf/454/6
